src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -35,8 +35,10 @@ private Configuration() {
         input = new FileInputStream(new File("config.yml"));
       } catch (FileNotFoundException ex) {
         try {
+//          LOG.info("CANNOT FIND FILE CONFIG.YML", ex);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)